src/components/AvatarGroup/index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Avatar } from '../';
+import StyledMoreAvatar from './styled/moreAvatar';
+import StyledContainer from './styled/container';
+
+/**
+ * Avatar groups are used to bunch together avatars
+ */
+export default function AvatarGroup(props) {
+    const { size, className, style, avatars, maxAvatars, showCounter } = props;
+
+    const spacing = '-6px';
+
+    const renderMoreItems = (max, total) => {
+        if (total <= max || !showCounter) return null;
+
+        const moreItems = avatars.slice(max).length;
+
+        return (
+            <StyledMoreAvatar style={{ zIndex: 0, marginLeft: `${spacing}` }}>
```

Comment:
Don't use styles inline, instead, let's define these styles in the styled component.